src/applications/bmqtool/m_bmqtool_application.h

```diff
@@ -167,6 +167,19 @@ class Application : public bmqa::SessionEventHandler {
     // message was seen during the current
     // grace period.
 
+    int d_numExpectedAcks;
+    // Auto-produce mode only. The total number of messages
+    // the tool will send. After posting is finished
+    // the tool will be waiting for this number of ACK
+    // messages, after which the shutdown semaphore will
+    // be posted.
+
+    int d_numPostedAcknowledged;
```

Comment:
Why is this 'posted'? It looks like an ACK counter, regardless of what is posted.  Or does it correlate GUIDs to make sure it is a valid ACK?

Reply:
There is nothing.  So, the "posted' part does not carry much meaning, does it.  It is just a bit confusing because one can think we have posted ACKs.
Very minor